### ISSUE TITLE
Inaccurate Capitalisation for Field Name

### ISSUE THREAD

**Issue by herrowna** [directus/format-title#11]

**Comment by rijkvanzanten** [comment]
This is technically working as expect, as the change from `_` to `F` indicates a new "word", the change from `Q` to lowercase `s` indicates another word, and the string `FAQs` isn't triggered as a special case (unlike `FAQ`, which is listed)

**Comment by udf2457** [comment]
Commenting here because #15 was an inadvertent duplicate, so discussion best continued here.

In #15, @rijkvanzanten suggested the translation "hack" as the fix for this.

The problem is that Directus does not handle translation fallback well (at all ?).

Namely, if I set my translation in `en-US` and my user is logged in under, e.g `en-GB`, the Directus UI does not fall-back to en-US as the default when no `en-GB` specific translation is available .

**Comment by rijkvanzanten** [comment]
> Commenting here because https://github.com/directus/format-title/issues/15 was an inadvertent duplicate, so discussion best continued here.

Nope, not inadvertent. See [my comment there](https://github.com/directus/format-title/issues/15#issuecomment-1218109324).

> Hence I don't really think the translation work-around is a sustainable fix. It has too many dependencies and conditions for it to work.

Feel free to [open a new Discussion on the directus/directus repo](https://github.com/directus/directus/discussions/new) outlining how you think it should work ideally. That's a better place to discuss this.

**Comment by rijkvanzanten** [comment]
Linear: [ENG-97](https://linear.app/directus/issue/ENG-97/render-related-faqs-as-related-faqs)
